docs/development/glossary/index.rst

```diff
@@ -53,6 +53,14 @@ or Client OS be used instead.
 minidump_stackwalk program are stored. The actual files are stored
 with a .jsonz extension.
 
+**Priority job**: A processing request (similar ``Reprocessing``) that has
+higher priority. The order of processing across all queues is: Priority,
+Standard, Reprocess, Priority. One crash from each at a time.
+Then it reverses the order and goes through those four again.
+And reverses again. Rinse and repeat. Priority jobs are processing of
+raw crashes that have never been processed. Crashes are put automatically
+on the Priority job queue based on this fact.
```

Comment:
replace the last sentence with:

Crashes are put on this queue only when a processed crash is requested from the UI and the associated raw crash has not been processed in the past.